docs/src/props-table.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import {Box, Link, Label} from '@primer/components'
+import Table from '@primer/gatsby-theme-doctocat/src/components/table'
+
+function PropsTable({children}) {
+  return (
+    <Table>
+      <thead>
+        <tr>
+          <Box as="th" textAlign="left">
+            Name
+          </Box>
+          <Box as="th" textAlign="left">
+            Type
+          </Box>
+          <Box as="th" textAlign="left">
+            Default
+          </Box>
+          <Box as="th" textAlign="left">
+            Description
+          </Box>
+        </tr>
+      </thead>
+      <tbody>{children}</tbody>
+    </Table>
+  )
+}
+
+function Row({name, type, defaultValue, description, required}) {
+  return (
+    <tr>
+      <Box as="td" fontFamily="mono" fontSize={1} fontWeight="bold" sx={{whiteSpace: 'nowrap'}}>
+        {typeof name === 'function' ? name() : name}
```

Comment:
When would `name` be a function?

Reply:
I was thinking maybe for an `(all DOM props)` row, we might format that name in that row without the monospaced font.